src/main/java/org/broadinstitute/hellbender/utils/python/PythonScriptExecutor.java

```diff
@@ -115,13 +115,9 @@ public boolean executeModule(final String moduleName, final List<String> pythonP
      */
     public boolean executeScript(final Resource scriptResource, final List<String> pythonProcessArgs, final List<String> scriptArgs) {
         Utils.nonNull(scriptResource, "script resource cannot be null");
+        // this File is automatically scheduled for deletion on exit
         final File tempResourceFile = IOUtils.writeTempResource(scriptResource);
-
-        try {
-            return executeScript(tempResourceFile.getAbsolutePath(), pythonProcessArgs, scriptArgs);
-        } finally {
-            FileUtils.deleteQuietly(tempResourceFile);
```

Comment:
Might want to consider keeping this -- it doesn't hurt to delete early when possible, despite the delete on exits.